examples/client/index.html

```diff
@@ -1,134 +1,316 @@
-
-        const query = db.query(
-          JSON.parse(editor.getValue())
-        );
-
-        bindQuery(query);
-      });
-
-      async function bindQuery(query) {
-        document.getElementById("next").disabled = true;
-
-        for (let i = 0; i < 10; i++) {
-          const result = await query.next();
-          if (result.done) {
-            return;
-          }
-          document.getElementById("results").innerHTML +=
-            JSON.stringify(result.value) + "\n";
-        }
-        while (true) {
-          document.getElementById("next").disabled = false;
-          await new Promise(
-            (resolve) => (document.getElementById("next").onclick = resolve)
-          );
-          document.getElementById("next").disabled = true;
-          const result = await query.next();
-          if (result.done) {
-            return;
-          }
-          document.getElementById("results").innerHTML +=
-            JSON.stringify(result.value) + "\n";
-        }
-      }
-    </script>
-  </head>
-  <body>
-    <h1>
-      <a href="https://github.com/kevmo314/appendable">Appendable</a> - NYC
-      Green Cab Trip Data in 01/2023
-    </h1>
-    <div>
-      Download the raw data here:
-      <a href="green_tripdata_2023-01.jsonl">JSONL</a> -
-      <a href="green_tripdata_2023-01.jsonl.index">Appendable Index</a> -
-      <a href="https://www.nyc.gov/site/tlc/about/tlc-trip-record-data.page"
-        >Source</a
-      >
-    </div>
-    <p>
-      Appendable is querying the JSONL and index files that GitHub pages hosts
-      <em>directly</em>. There is no server involved here!
-    </p>
-    <p>
-      Keep in mind that while the query syntax supports a lot of different
-      operations, Appendable doesn't support composite indexes yet. Therefore,
-      only one field at a time can be filtered on and that field must be used
-      for sorting.
-    </p>
-    <div style="display: flex">
-      <div>
-        <h2>Fields</h2>
-        <pre id="fields"></pre>
-      </div>
-      <div>
-        <h2>Query</h2>
-        <div id="json-editor" style="height: 300px; width: 600px;"></div>
-        <button id="execute">Execute</button>
-        <h2>Results</h2>
-        <pre id="results"></pre>
-        <button id="next">Fetch more</button>
-      </div>
-    </div>
-    <script>
-      var editor = ace.edit("json-editor");
-      editor.session.setMode("ace/mode/json");
-      editor.setTheme("ace/theme/chrome");
-      editor.getSession().setTabSize(2);
-      editor.getSession().setUseSoftTabs(true);
-
-      editor.setValue(JSON.stringify({
-        "where": [
-          {
-            "operation": ">=",
-            "key": "trip_distance",
-            "value": 10
-          }
-        ],
-        "orderBy": [
-          {
-            "key": "trip_distance",
-            "direction": "ASC"
-          }
-        ]
-      }, null, 2), -1);
-    </script>
-  </body>
+	<head>
```

Comment:
Can you format this with `prettier`?

Reply:
So my prettier says `index.html` has been formatted:

```
["INFO" - 10:17:20 PM] File Info:
{
  "ignored": false,
  "inferredParser": "html"
}
["INFO" - 10:17:20 PM] No local configuration (i.e. .prettierrc or .editorconfig) detected, falling back to VS Code configuration
["INFO" - 10:17:20 PM] Prettier Options:
{
  "arrowParens": "always",
  "bracketSpacing": true,
  "endOfLine": "lf",
  "htmlWhitespaceSensitivity": "css",
  "insertPragma": false,
  "singleAttributePerLine": false,
  "bracketSameLine": false,
  "jsxBracketSameLine": false,
  "jsxSingleQuote": false,
  "printWidth": 80,
  "proseWrap": "preserve",
  "quoteProps": "as-needed",
  "requirePragma": false,
  "semi": true,
  "singleQuote": false,
  "tabWidth": 2,
  "trailingComma": "es5",
  "useTabs": true,
  "embeddedLanguageFormatting": "auto",
  "vueIndentScriptAndStyle": false,
  "filepath": "/Users/matthew/Desktop/GARAGE/appendable/examples/client/index.html",
  "parser": "html"
}
["INFO" - 10:17:20 PM] Formatting completed in 41ms.
```